modules/configuration-api/src/main/java/org/apache/ignite/configuration/ConfigurationListenOnlyException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.configuration;
+
+/**
+ * This exception is used if an attempt was made to get/update a configuration value in listen-only mode.
+ */
+public class ConfigurationListenOnlyException extends RuntimeException {
```

Comment:
```suggestion
public class ConfigurationListenOnlyException extends IgniteException {
```

Reply:
Then all the `Configuration*Exception` need to be done this way, I think we need to ask @ibessonov opinion.

Reply:
Let's leave it as it is.